src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> STORAGE = new HashMap<>();
```

Comment:
It is not a constant because constant means it does not change, but you can add or remove values ​​from the map, so it is modifiable and can be a constant.
Check Collection.emptyMap() or Map.of() methods. They return an immutable map
```suggestion
    public static final Map<String, Integer> storage = new HashMap<>();
```